File: 백준/Bronze/9063. 대지/대지.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').toString().trim().split('\n');
const N = +input[0];
const coords = input.slice(1).map(r => r.split(' ').map(Number));
let top = coords[0][1];
let bottom = coords[0][1];
let left = coords[0][0];
let right = coords[0][0];

for (let [x, y] of coords) {
    top = Math.max(top, y);
    bottom = Math.min(bottom, y);
    left = Math.min(left, x);
    right = Math.max(right, x);
}

console.log((top - bottom) * (right - left));

File: 백준/Bronze/9063. 대지/대지.py
import sys

N = int(input())
M = [list(map(int,row.split())) for row in sys.stdin.readlines()]
min_x = 10000
min_y = 10000
max_x = -10000
max_y = -10000
for x,y in M:
    min_x = min(x, min_x)
    min_y = min(y, min_y)
    max_x = max(x, max_x)
    max_y = max(y, max_y)
    
W = max_x - min_x
H = max_y - min_y
print(W * H)